src/main/java/org/testng/reporters/EmailableReporter2.java

```diff
@@ -41,9 +41,20 @@
     // Reusable buffer
     private StringBuilder buffer = new StringBuilder();
 
+    private String fileName = "emailable-report.html";
```

Comment:
There's a setter via which this field can be altered. So this cannot be changed into a `final` variable.